modules/installation-aws-config-yaml.adoc

```diff
@@ -17,6 +24,7 @@ This sample YAML file is provided for reference only. You must obtain your
 
 [source,yaml]
 ----
+ifdef::without-networking[]
```

Comment:
At the time, including a number annotation for networking parameters didn't seem to make sense in the context of the non-network customization path for AWS installation.

Reply:
Because _2_ says: _If you do not provide these parameters and values, the installation program provides the default value._ But you cannot provide those values using the installation path for **Installing a cluster on AWS with customizations**.